tests/sentry_plugins/sessionstack/test_plugin.py

```diff
@@ -9,7 +9,7 @@
 
 EXPECTED_SESSION_URL = (
     "https://app.sessionstack.com/player/#/sessions/588778a6c5762c1d566653ff"
-    "?access_token=example-access-token&source=sentry"
+    "?source=sentry&access_token=example-access-token"
```

Comment:
Can we revert this change?

Reply:
I tried reverting it, however, the tests fail.
I believe that moving to Python 3 installs a different dependency for whatever handles URL query parameters.
It seems to me only a change in order of parameters.